react-validated-input-component/src/validated-input.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+
+class Password extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      status: false,
+      password: '',
+      message: 'A password is required'
+    };
+
+    this.handleChange = this.handleChange.bind(this);
+
+  }
+
+  handleChange(event) {
+    const value = event.target.value;
+    this.setState({ password: value });
+    const password = event.target.value;
```

Comment:
It is redundant to assign the same value to two different variables, you could do this instead:
```JSX
const password = event.target.value;
this.setState({ password: password });
```